src/components/HeaderBarCustomDialog.vue

```diff
@@ -209,54 +207,37 @@ watch(
   }
 );
 
-const applyDefaultSetting = () => {
-  $q.dialog({
+const applyDefaultSetting = async () => {
+  const result = await store.dispatch("SHOW_CONFIRM_DIALOG", {
     title: "ツールバーをデフォルトに戻します",
     message: "ツールバーをデフォルトに戻します。<br/>よろしいですか？",
     html: true,
-    ok: {
-      label: "はい",
-      flat: true,
-      textColor: "display",
-    },
-    cancel: {
-      label: "いいえ",
-      flat: true,
-      textColor: "display",
-    },
-  }).onOk(() => {
+    actionName: "はい",
+    cancel: "いいえ",
+  });
+  if (result === "OK") {
     toolbarButtons.value = [...defaultSetting];
     selectedButton.value = toolbarButtons.value[0];
-  });
+  }
 };
 const saveCustomToolbar = () => {
   store.dispatch("SET_TOOLBAR_SETTING", {
     data: [...toolbarButtons.value],
   });
 };
 
-const finishOrNotDialog = () => {
+const finishOrNotDialog = async () => {
   if (isChanged.value) {
-    $q.dialog({
+    const result = await store.dispatch("SHOW_CONFIRM_DIALOG", {
```

Comment:
ここはちょっと間違えて閉じようとしちゃう人もいそうなのと、結構再設定に時間かかりそうなのでWARNINGでも良いかも？
```suggestion
    const result = await store.dispatch("SHOW_WARNING_DIALOG", {
```

Reply:
デフォルトのフォーカスがconfirmがOK、WarningがCancelとなったので元が`focus: "cancel",`だった箇所はWARNINGに置き換えるのが妥当そうですね！他にも2箇所あったのでやっておきました